src/io.rs

```diff
@@ -22,14 +26,25 @@ where
     };
 }
 
+pub fn write_protobuf<I: Interfaceable>(
+    src: &mut I,
+    game: Game,
+) -> Result<Game, String> {
+    let mut file = save(src, &game);
+    let mut cos = CodedOutputStream::new(&mut file);
+    game.write_to(&mut cos).unwrap();
+    cos.flush().unwrap();
+    return Ok(game);
+}
+
 pub fn prompt_path<F, M: MessageStatic, I: Interfaceable>(
     src: &mut I,
```

Comment:
I think we could probably rework this into screen.rs and call from `fn save` as well

Reply:
I did not do this but I did all the other things